### DISCUSSION TITLE
Rate limited

### DISCUSSION THREAD

**Discussion by Johnwulp** [shibayan/appservice-acmebot#369] [Q&A] [answered]
Hi,

In our Azure resource group, we have around 8 applications.
For 7 I can create successfully certificates, for number 8 I see in the application insight logging that a result code 429 Is received after the new-order request.

How can I view the actual response for this request?
![image](https://user-images.githubusercontent.com/879840/140774654-43c44fb7-2b7a-428d-a4d5-bb1f4819b36b.png)

**Comment by shibayan** [comment, marked as answer]
The error code of the response should be included in the exception message, but if it is not there, there is no way to check it.

You will need to refer to the Let's Encrypt rate limiting documentation.
https://letsencrypt.org/docs/rate-limits/

**Comment by Johnwulp** [comment]
Ok, I will retry tomorrow.
For reference the error is not visible in the alert box or should it be visible somewhere else?

<img width="464" alt="image" src="https://user-images.githubusercontent.com/879840/140805334-7826a92d-b5de-4c4a-8936-eddcb436ea66.png">
